### PR TITLE
change term "component" to "service" in commands 

### PR DESCRIPTION
**Pull request by captainroy-hy**
related to #412 
This PR changed term "component" in CLI (usage and outputs) to "service". 

Signed-off-by: roy wang <seiwy2010@gmail.com>

### CODE REVIEW

File: pkg/commands/init.go [hunk 1]
```diff
@@ -147,12 +147,12 @@ func (o *appInitOptions) Workload() error {
 		workloadList = append(workloadList, w.Name)
 	}
 	prompt := &survey.Select{
-		Message: "Choose an workload for your component: ",
+		Message: "Choose Service type: ",
```
Comment (wonderflow) [inline pkg/commands/init.go:150]:
```suggestion
		Message: "Choose workload type for your service: ",
```

File: pkg/commands/init.go [hunk 2]
```diff
@@ -147,12 +147,12 @@ func (o *appInitOptions) Workload() error {
...
 		Options: workloadList,
 	}
 	err = survey.AskOne(prompt, &o.workloadType)
 	if err != nil {
-		return fmt.Errorf("read workload type err %v", err)
+		return fmt.Errorf("read Service type err %v", err)
```
Comment (wonderflow) [inline pkg/commands/init.go:155]:
```suggestion
		return fmt.Errorf("read workload type err %v", err)
```

File: pkg/commands/status.go [hunk 3]
```diff
@@ -72,7 +72,7 @@ const (
 
 const (
 	ErrNotLoadAppConfig  = "cannot load the application"
-	ErrFmtNotInitialized = "oam-core-controller cannot initilize the component: %s"
+	ErrFmtNotInitialized = "KubeVela runtime cannot initilize the service: %s"
```
Comment (wonderflow) [inline pkg/commands/status.go:75]:
```suggestion
	ErrFmtNotInitialized = "KubeVela core cannot initilize the service: %s"
```

File: pkg/commands/status.go [hunk 4]
```diff
@@ -535,9 +535,9 @@ func trackHealthCheckingStatus(ctx context.Context, c client.Client, compName, a
 		healthStatus = HealthStatusNotDiagnosed
 		statusInfo, _, _, err := getWorkloadInstanceStatusAndCreationTime(ctx, c, env.Namespace, wlStatus.Reference)
 		if err != nil {
-			return compStatusUnknown, HealthStatusUnknown, "", errors.Wrap(err, "WARN: The component type is unknown to HealthScope and cannot get status.")
+			return compStatusUnknown, HealthStatusUnknown, "", errors.Wrap(err, "WARN: The Service type is unknown to HealthScope and cannot get status.")
 		}
-		healthInfo := fmt.Sprintf("WARN: The component type is unknown to HealthScope.\nYou may check component status with [%s/%s] status: \n%s",
+		healthInfo := fmt.Sprintf("WARN: The Service type is unknown to HealthScope.\nYou may check Service status with [%s/%s] status: \n%s",
```
Comment (wonderflow) [inline pkg/commands/status.go:540]:
```suggestion
		healthInfo := fmt.Sprintf("WARN: The Service type is unknown to HealthScope.\nYou may check service status with [%s/%s] status: \n%s",
```

**Review by wonderflow — APPROVED** [review]
LGTM, can you please also refresh the doc

**Comment by captainroy-hy** [comment]
> LGTM, can you please also refresh the doc

Got it. I will refresh the doc.

**Pull request merged**